devicemap-schema.json

```diff
@@ -88,6 +95,9 @@
               "vmbusId": {
                 "type": "string",
                 "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$"
+              },
+              "parent": {
+                "type": "null"
```

Comment:
Does this ensure that it is NOT specified for these other cases?

Reply:
Yes, but it also allows the vmbusId and pciAddress keys to be present, but the values to be `None`. I believe this was necessary for some of the dynamic reconfig stuff